README.md

```diff
@@ -179,9 +180,62 @@ In order to only do inserts for example, use --skip-deletes and --skip-updates f
 `pg_comparator --verbose --do-it --synchronize --skip-deletes --skip-updates pgsql://dataservices:insecure@dso_database/dataservices/table_one pgsql://dataservices:insecure@dso_database/dataservices/table_two`
 
 The second example below compares the table 'table_one' with 'table_two' on the different/seperate database.
-Beware! Both databases must have the libraries installed and the pgcmp extension created. 
+Beware! Both databases must have the libraries installed and the pgcmp extension created.
 `pg_comparator --verbose --do-it --synchronize pgsql://dataservices:insecure@dso_database/dataservices/table_one pgsql://ds_airflow:insecure@database/ds_airflow/table_two`
 
 The third example below compares the table 'table_one' with 'table_two' on the different/seperate database, but with the extra argument of looking at specific columns. These are indicated by adding a questionmark (?) following the key of the source table, then a semicolon (:) following the columns to use in the comparision seperated by a comma.
-Beware! Both databases must have the libraries installed and the pgcmp extension created. 
+Beware! Both databases must have the libraries installed and the pgcmp extension created.
 `pg_comparator --verbose --do-it --synchronize pgsql://dataservices:insecure@dso_database/dataservices/table_one?column1:column2,column3 pgsql://ds_airflow:insecure@database/ds_airflow/table_two`
+
+# PyCharm
+
+PyCharm has a great source code formatter that is extremely fast.
+But not everyone uses PyCharm.
+To ensure Python code is formatted consistently,
+regardless of the development environment used,
+we use `black` and `isort`.
+PyCharm can easily be configured to run these those external programs.
+First create a shell script `fmt_code.sh`:
+
+
+```sh
+#!/usr/bin/env sh
+
+PY_INTERPRETER_PATH=$1
+FILE_PATH=$2
+
+${PY_INTERPRETER_PATH} -m black ${FILE_PATH}
+${PY_INTERPRETER_PATH} -m isort ${FILE_PATH}
+```
```

Comment:
Kunnen de tools niet direct vanuit pycharm draaien? De variabelen haal je immers ook in de run config naar boven.

Reply:
Oh, sure, maar ik wil beide tools met één keyboard shortcut draaien. Daarom heb ik ze opgenomen in een shell script.